Module.php

```diff
@@ -405,7 +421,7 @@ public function generateHash($key, $userGuid, $isEmpty = false)
     public function readHash($hash)
     {
         try {
-            $data = JWT::decode($hash, Yii::$app->settings->get('secret'), array('HS256'));
```

Comment:
I'm not sure this should have been removed, at most `getJwtAlgorithm()` should also be used here.